source/brailleDisplayDrivers/handyTech.py

```diff
@@ -524,6 +525,20 @@ class Modular80(Modular):
 	numCells = 80
 
 
+class Activator(TimeSyncFirmnessMixin, AtcMixin, JoystickMixin, TripleActionKeysMixin, Model):
+	deviceId = MODEL_ACTIVATOR
+	numCells = 40
+	genericName = name = 'Activator'
+
+	def _get_keys(self):
+		keys = super(Activator, self).keys

```

Comment:
I believe this notation is more standard for modern python

```suggestion
		keys = super().keys
```